lib/infrastructure/providers/my_user_provider/my_user_page_view_model.dart

```diff
@@ -38,60 +89,41 @@ class MyUserPageViewModel {
     TokenModel? secondUserTokenModel;
 
     firstUser = bidIn.user;
-    if (firstUser == null) {
-      return false;
-    }
-
-    firstUserTokenModel = await database.getTokenFromId(firstUser.id);
+    firstUserTokenModel = await database.getTokenFromId(firstUser!.id);
 
     if (bidIns.length > 1) {
       secondUser = bidIns[1].user;
       secondUserTokenModel = await database.getTokenFromId(secondUser!.id);
     }
 
-    if (!bidIn.public.active) return false;
+    bool camera = true;
+    bool microphone = true;
 
-    if (bidIn.user!.isInMeeting()) {
-      await cancelNoShow(bidIn: bidIn);
-      return false;
+    if (!kIsWeb) {
+      camera = await Permission.camera.request().isGranted;
```

Comment:
what if user does not want to share camera ~ we allow muting, so we should not require permission for camera to start
solve later

Reply:
Then we should have to think of scenario that 

if user pay host for video call and host turn off video then?

Reply:
Host can always turn video off...e.g. by putting tape over their camera
If a feature is unavoidable better to make it easy

Ringing page needs to have audio or video muting as well... Does it not?